### PR TITLE
Update README.md

### LINKED ISSUE #16: The command in README need to be modified
`
ifconfig | grep "inet " | grep -v 127.0.0.1 | awk '{ print $2 }
`

I 'm running this command in Ubuntu 16.04 , the result is

`addr:172.17.0.1 `
 `addr:192.168.247.129`.

It doesn't just contain ip.

### PR DESCRIPTION
**Pull request by fxbing**
fixed #16

### CODE REVIEW

**Comment by sijie** [comment]
@Jennifer88huang @Anonymitaet please help review this pull request.

File: README.md [hunk 1]
```diff
@@ -56,6 +56,16 @@ $ STANDALONE_HOST="$(ifconfig | grep "inet " | grep -v 127.0.0.1 | awk '{ print
...
 ```
 
+In Ubuntu, it should be
+```bash
+
+$ STANDALONE_HOST="$(ifconfig | grep "inet " | grep -v 127.0.0.1 | awk '{ print $2 }' | awk -F ':' '{ print $2 }' | awk 'NR==2')" j2 prometheus/standalone.yml.template > /tmp/standalone.prometheus.yml
+
+```
+
+If it still doesn't work properly, I suggest setting the IP manually.
```
Comment (Jennifer88huang) [inline README.md]: It is grammatically correct, we can refine it to be more concise and avoid using "suggest"/"recommend" in technical writing. The following refinement is for your reference. 
"If it does not work properly, you can set the IP manually."
If you adopt the suggestion, please check all similar cases.

File: README.md [hunk 2]
```diff
@@ -77,6 +87,14 @@ export PULSAR_CLUSTER=standalone
 docker run -it -p 3000:3000 -e PULSAR_PROMETHEUS_URL="${PULSAR_PROMETHEUS_URL}" -e PULSAR_CLUSTER="${PULSAR_CLUSTER}" streamnative/apache-pulsar-grafana-dashboard:latest 
 ```
 
+In Ubuntu, you should set `PULSAR_PROMETHEUS_URL` in this way.
```
Comment (Jennifer88huang) [inline README.md]:
```suggestion
In Ubuntu, set `PULSAR_PROMETHEUS_URL` in this way.
```

File: README.md [hunk 3]
```diff
@@ -77,6 +87,14 @@ export PULSAR_CLUSTER=standalone
...
+
+```bash
+export PULSAR_PROMETHEUS_URL=http://$(ifconfig | grep "inet " | grep -v 127.0.0.1 | awk '{ print $2 }' | awk -F ':' '{ print $2 }' | awk 'NR==2'):9090
+```
+
+If it still doesn't work properly, I suggest setting the IP manually.  
+
 Access the Grafana Dashboard at http://localhost:3000.
 The default user name and password are `admin` and `happypulsaring`. It is set in the [conf/grafana.ini](conf/grafana.ini) file.
```
Comment (Anonymitaet) [inline README.md]:
```suggestion
The default user name is `admin`, the default password is `happypulsaring`, and they are set in the [conf/grafana.ini](conf/grafana.ini) file.
```

**Comment by fxbing** [comment]
Thanks for your review. @sijie @Jennifer88huang @Anonymitaet 
I have made changes based on comments.

File: README.md [hunk 4]
```diff
@@ -56,6 +56,16 @@ $ STANDALONE_HOST="$(ifconfig | grep "inet " | grep -v 127.0.0.1 | awk '{ print
 
 ```
 
+In Ubuntu, it is
```
Comment (Anonymitaet) [inline README.md]:
```suggestion
In Ubuntu, set `STANDALONE_HOST` as below.
```

**Pull request merged**